code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -169,6 +169,36 @@
 	item_state = "cardborg"
 	flags_armor_protection = UPPER_TORSO|LOWER_TORSO
 	flags_inv_hide = HIDEJUMPSUIT
+/* not currently needed thanks to Manzehino who provided a different jumpsuit sprite.
```

Comment:
You could still add it in the code but as admin-only alternative spawn if it's distinct enough.